### ISSUE TITLE
Cell types for a specific organism show results on the heatmap across all organisms

### ISSUE THREAD

**Issue by pcm32** [ebi-gene-expression-group/atlas-web-single-cell#208]
When querying https://wwwdev.ebi.ac.uk/gxa/sc/search/metadata/neuron and clicking on any of the Drosophila Melanogaster "neuron" cell types, you get results from experiments across different organisms (for instance E-CURD-52 is mouse).

![image](https://user-images.githubusercontent.com/368478/137144846-ec9b7527-5618-4dc0-8930-041c9410ca4e.png)

Shouldn't we limit that per organism, so that if clicking on the Drosophila neuron, we only get fly studies on the heatmap?

**Comment by alfonsomunozpomer** [comment]
Yes, the controller that returns the heatmap JSON payload admits the experiment accessions and the wheel contains the necessary information, but we’re not sending it as part of the request. Thanks for letting us know.
